content/docs/apidocs/config/v1alpha1.md

```diff
@@ -45,6 +45,46 @@ string
 </tr>
 </tbody>
 </table>
+<h3 id="config.openservicemesh.io/v1alpha1.ClusterSpec">ClusterSpec
+</h3>
+<p>
+(<em>Appears on:</em><a href="#config.openservicemesh.io/v1alpha1.MultiClusterServiceSpec">MultiClusterServiceSpec</a>)
```

Comment:
I am inclined to not document the multicluster spec yet, given it's not ready and documenting it on the website is going to cause confusion.

Can we remove the multicluster def?

Reply:
+1

Reply:
Good point, thanks!